### ISSUE TITLE
Platypus: Home link missing discernible text

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/platypus#472]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, the page's home link (icon at the top left) is missing a discernible text.

---

https://dequeuniversity.com/rules/axe/4.3/link-name?application=axeAPI

---

**User impact:** Serious
**Disabilities affected:** Blind, Deafblind, Mobility
**Requirement(s):** Section 508: MUST
**WCAG Success Criteria:** 2.4.4 Link Purpose (In Context), 4.1.2 Name, Role, Value

**Comment by micpap25** [comment]
Hi, would this be "discernable text"? Is this different from how the home icon used to be? This seems like a resolved issue to me.
<img width="141" alt="image" src="https://user-images.githubusercontent.com/34077605/212341382-e5000b7b-6ed3-4fb7-a6a6-61ecbaf5a5d7.png">

**Comment by vabarbosa** [comment]
This repository is no longer maintained. It has been superseded by [IBM Quantum Learning](https://learning.quantum.ibm.com).
